### ISSUE TITLE
[HELP NEEDED] AIR Mobile/Desktop Output

### ISSUE THREAD

**Issue by angelhdz** [apache/royale-asjs#522]
### Please follow this Issue

https://github.com/apache/royale-asjs/issues/229

I've posted there everything I could about my files and environment, screenshots, listed the environment variable paths, etc.

JS/HTML output and SWF (Flash Player Debugger) output works 100%, but AIR Mobile/Desktop output doesn't.  I followed the royale SDK README.md file steps one by one, but I don't find how to set up VSCode launch.json and tasks.json and asconfig.json to allow AIR output.

I'm trying to learn Royale so I can get others to try it , but I've had a hard time with different issues and the help is little, the documentation is poor.

I even contributed by making videos on YouTube on how to configure a royale workspace and starting files, but only for JS output and I know actionscripters are gonna ask me about SWF/AIR output soon.

One of the members of my Actionscript 3.0 Discord Server asked me "Is Royale ELI5 (Explain Like I'm 5) and I had to sadly say "No".

I'm trying Royale for 3rd time, to see if the bugs I reported were fixed in the latest update but I'm starting to give up, so please, if you have time, give me a hand. Thanks, cheers

**Comment by aharui** [comment]
Hi Angel,

I think you meant https://github.com/apache/royale-asjs/issues/522?  Not #229.

@Josh Tynjala<mailto:joshtynjala@apache.org> can you find some time to work with Angel.  AIUI, he’s having trouble getting AIR configured in VSCode.

-Alex

**Comment by angelhdz** [comment]
@aharui I referred to #229 because I provided so much info there, screenshots, configuration files contents, etc. instead of having to post everything here. Makes sense?

**Comment by joshtynjala** [comment]
@angelhdz 

For reference, here is the asconfig.json file that you posted in #229:

```json5
{
    
    "compilerOptions": {
        "debug":  false,
        "source-map": true,
        "targets": [
            "JSRoyale","SWF"
        ],
        "theme": "${royalelib}/themes/JewelTheme/src/main/resources/defaults.css",       
        "source-path": [
            "src"
        ]
    },
    "files": [
        "src/HelloRoyale.mxml"
    ],
}
```

This file is not configured correctly for Adobe AIR. I see a few things that you need to do:

1. Set the "config" field to either "air" or "airmobile".
2. Set the "application" field to an *-app.xml* application descriptor file.
3. Remove "JSRoyale" from "targets". It should contain "SWF" only. (This one may not be strictly necessary, and you should be able to add it back later, but I'd recommend starting with a simpler build process to avoid too much complexity).

I should note that I've never tried using Royale with Adobe AIR. As far as I know, no one else has mentioned trying it either. Even once we get things configured correctly, we may run into some bugs that I'll need to fix in the VSCode extension because I've focused my time developing the VSCode extension exclusively with projects that generate JS only.
